articles/azure-resource-manager/management/move-support-resources.md

```diff
@@ -1453,55 +1453,55 @@ Jump to a resource provider namespace:
-> | loadbalancers | Yes - Basic SKU<br> Yes - Standard SKU | Yes - Basic SKU<br>No - Standard SKU | Yes <br/><br/> Use [Azure Resource Mover](../../resource-mover/tutorial-move-region-virtual-machines.md) to move internal and external load balancers. |
-> | localnetworkgateways | Yes | Yes | No |
+> | loadbalancers | **Yes** - Basic SKU<br> **Yes** - Standard SKU | **Yes** - Basic SKU<br>No - Standard SKU | **Yes** <br/><br/> Use [Azure Resource Mover](../../resource-mover/tutorial-move-region-virtual-machines.md) to move internal and external load balancers. |
+> | localnetworkgateways | **Yes** | **Yes** | No |
 > | natgateways | No | No | No |
 > | networkexperimentprofiles | No | No | No |
-> | networkintentpolicies | Yes | Yes | No |
-> | networkinterfaces | Yes | Yes | Yes <br/><br/> Use [Azure Resource Mover](../../resource-mover/tutorial-move-region-virtual-machines.md) to move NICs. |
+> | networkintentpolicies | **Yes** | **Yes** | No |
+> | networkinterfaces | **Yes** | **Yes** | **Yes** <br/><br/> Use [Azure Resource Mover](../../resource-mover/tutorial-move-region-virtual-machines.md) to move NICs. |
 > | networkprofiles | No | No | No |
-> | networksecuritygroups | Yes | Yes | Yes <br/><br/> Use [Azure Resource Mover](../../resource-mover/tutorial-move-region-virtual-machines.md) to move network security groups (NSGs). |
+> | networksecuritygroups | **Yes** | **Yes** | **Yes** <br/><br/> Use [Azure Resource Mover](../../resource-mover/tutorial-move-region-virtual-machines.md) to move network security groups (NSGs). |
 > | networkwatchers | No | No | No |
-> | networkwatchers / connectionmonitors | Yes | No | No |
-> | networkwatchers / flowlogs | Yes | No | No |
-> | networkwatchers / pingmeshes | Yes | No | No |
+> | networkwatchers / connectionmonitors | **Yes** | No | No |
+> | networkwatchers / flowlogs | **Yes** | No | No |
+> | networkwatchers / pingmeshes | **Yes** | No | No |
 > | p2svpngateways | No | No | No |
-> | privatednszones | Yes | Yes | No |
-> | privatednszones / virtualnetworklinks | Yes | Yes | No |
+> | privatednszones | **Yes** | **Yes** | No |
+> | privatednszones / virtualnetworklinks | **Yes** | **Yes** | No |
 > | privatednszonesinternal | No | No | No |
 > | privateendpointredirectmaps | No | No | No |
-> | privateendpoints | Yes - for [supported private-link resources](./move-limitations/networking-move-limitations.md#private-endpoints)<br>No - for all other private-link resources | Yes - for [supported private-link resources](./move-limitations/networking-move-limitations.md#private-endpoints)<br>No - for all other private-link resources | No |
+> | privateendpoints | **Yes** - for [supported private-link resources](./move-limitations/networking-move-limitations.md#private-endpoints)<br>No - for all other private-link resources | **Yes** - for [supported private-link resources](./move-limitations/networking-move-limitations.md#private-endpoints)<br>No - for all other private-link resources | No |
```

Comment:
If there are limitations, could the cells read "_Limited_" or "_Partly_" instead of getting readers' hopes up with "Yes...but"?

Reply:
I think that's a good idea. I would lean towards "Limited" with the link to the list of supported types.

Reply:
I just went through the whole document to look for other cells that would benefit from being changed from "Yes" to "Limited" but AFAICT it's just these 2, so probably not worth making a special-case here.